.github/workflows/maven-tests.yml

```diff
@@ -14,9 +14,12 @@ jobs:
     - name: Set up Java
       uses: actions/setup-java@v2
       with:
-        java-version: '11'
+        java-version: '17'
         distribution: 'adopt'
 
+    - name: Set JAVA_HOME
+      run: export JAVA_HOME=/usr/lib/jvm/jre-17-openjdk
```

Comment:
I tried installing `openjdk-17-jdk` on a Debian container, the files got installed under `/usr/lib/jvm/java-17-openjdk-amd64` so maybe try that instead.

Reply:
Can we actually set an environment variable this way in GH workflow? Or do we need to use `env:`?

Reply:
The SonarCloud now is working but the env with the correct jvm should go in all tests with ubuntu/debian.